app/gui/src/project-view/components/ComponentBrowser/ComponentEditor.vue

```diff
@@ -79,8 +79,14 @@ const rootStyle = computed(() => {
     <div :class="{ componentEditorIcon: true, port: props.mode.mode !== 'componentBrowsing' }">
       <SvgIcon :name="icon" />
     </div>
-    <span class="selfArgInfo">{{ selfTypeName ?? 'Input' }} Components</span>
-    <SvgIcon class="selfArgInfoArrow" name="folder_closed" />
+    <span v-if="mode.mode === 'componentBrowsing'" class="selfArgInfo"
+      >{{ selfTypeName ?? 'Input' }} Components</span
```

Comment:
I think it's a bit confusing to display `Any Components` here--the user may not read `Any` as referring to a type, in which case it could seem like we meant `All Components`. It would help if we format the component type in monospace.

Reply:
This is how it looks: 

![image](https://github.com/user-attachments/assets/0bdde651-c693-4811-bd77-cab16aecc976)

People may think it's a bit ugly. @AdRiley what do you think?

Reply:
In case of Any:

![image](https://github.com/user-attachments/assets/62d25b90-2855-4ae8-8dc0-71b989db96da)

Reply:
After all we agreed, that, if type is unknown, we don't display the label at all. In that case, `Any` should be never visible.